src/java/org/apache/cassandra/db/compaction/CompactionThroughputThrottler.java

```diff
@@ -0,0 +1,71 @@
+package org.apache.cassandra.db.compaction;
+
```

Comment:
could you move this to a package called `com.palantir.cassandra.db.compaction`. This makes it easier to track changes in the fork.